docs/install.md

```diff
@@ -46,6 +46,12 @@ you can choose to build and run specific sections of Calypso using the `SECTION_
 
 For instance, `SECTION_LIMIT=reader,login npm start` would start Calypso and only build the `reader` and `login` sections.
 
+Additionally, in Calypso we use multiple [Webpack entry points](https://webpack.js.org/concepts/entry-points/) for separating concerns and serving smaller bundles to the user at any given time.
+Building a limited number of entry points speeds up the build process, and to allow that, the `ENTRY_LIMIT` environment variable is available to allow building and running only a specific entry point.
+
+For example: `ENTRY_LIMIT=entry-login,entry-main npm start` would start Calypso and only build the login and the main entry points.
+
+To find all the available enty points, you can refer to the `entry` option in Calypso's primary `webpack.config.js` file.
```

Comment:
Could we add a line like this to the section section as well? It's currently missing (and has some grammar goofs)

Reply:
Sure 👍 Added in the latest commits. Care to take another look and tell me if I'm missing something? Grammar isn't my strong side 😉

Reply:
LGTM!